### DISCUSSION TITLE
New Asus GUI

### DISCUSSION THREAD

**Discussion by Peke** [gnuton/asuswrt-merlin.ng#536] [Ideas]
On This page ASUS is finally working on VLAN supports for WAN  and WLANs any plans to work on this?

https://www.asus.com/us/support/faq/1050072/

https://www.asus.com/support/faq/1049415/

https://blog.flashrouters.com/2023/09/26/do-asus-routers-support-vlan-can-you-set-up-a-vlan-network-on-an-asus-router-which-asus-routers-support-vlan-setup/

https://virtualize.link/asus-vlans/

https://www.testipenkki.com/?p=52

**Comment by gnuton** [comment]
No plans yet, those are pro features and I am not sure if those will make these available to all routers if possible with the next big stock release.
If something will happen we will make it public

**Reply by rdegaud** [reply]
Can we get active by writing/grouping or otherwise to sensitize the asus team to implement this internet/vlan/802.1Q profile feature on products that interest us?

**Comment by SOBON2O** [comment]
I second this. I own an Asus RT-AX82U, and would love a GUI overhaul. VLANs would be a dream come true.

**Reply by kokosowy** [reply]
TUF-AX3000 v2 owner here and I’d also love to have this feature eventually. Is this development on ASUS side or its matter of implementation on Merlin’s? Would any bounty/donation help to get this feature?

**Reply by gnuton** [reply]
@kokosowy what kind of UI changes would you like to see? New theme?

**Reply by gnuton** [reply]
> I second this. I own an Asus RT-AX82U, and would love a GUI overhaul. VLANs would be a dream come true.

Let see if this will come later. But I do not think this will be available without big changes

**Reply by kokosowy** [reply]
> @kokosowy what kind of UI changes would you like to see? New theme?

Hi @gnuton! Sorry for potential confusion, I was talking about VLAN feature.
